ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+**Issue Template**
+&lt;Title&gt;: &lt;One line description&gt;
+
+*Description:* &lt;Describe the issue. Please be detailed.
```

Comment:
instead of the lt gt demarcation can we do something like this:

```
*Description*:
> Describe the issue. Please be detailed. If a feature request, please describe the desired behaviour, what 
scenario it enables and how it would be used.
```
which renders as

*Description*:
> Describe the issue. Please be detailed. If a feature request, please describe the desired behaviour, what 
scenario it enables and how it would be used.

Easier to read and edit when working with the raw markdown.

Reply:
That is nicer. Thank you for the suggestion. I'll update the templates with it.